### PR TITLE
First version for iac-blog

### PR DESCRIPTION
**Pull request by daianamezdrea**

### CODE REVIEW

File: _posts/2020-03-27-status-update.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+- [puppetlabs-postgresql#1156](https://github.com/puppetlabs/puppetlabs-postgresql/pull/1156) by [veninga](https://github.com/veninga)
+- [puppetlabs-firewall#908](https://github.com/puppetlabs/puppetlabs-firewall/pull/908) by [jbro](https://github.com/jbro)
+- [puppetlabs-postgresql#1158](https://github.com/puppetlabs/puppetlabs-postgresql/pull/1158) by [XnS](https://github.com/XnS)
+
+We look forward to see your new contributions to Puppet modules.
+
+## Customer Escalations
+
+This week, the team worked at Panos module. (check out this [PR](https://github.com/puppetlabs/puppetlabs-panos/pull/119) for more details). Our purpose is to unblock testing and progress development.
```
Comment (sanfrancrisko) [inline _posts/2020-03-27-status-update.md]:
```suggestion
This week, the team worked on the Panos module. (check out this [PR](https://github.com/puppetlabs/puppetlabs-panos/pull/119) for more details). Our purpose is to unblock testing and progress development.
```

File: _posts/2020-03-27-status-update.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+A bit further out and we'll have a new namespace on the forge - dsc - where we'll publish these automatically generated modules as 1:1 matches for the PowerShell modules with DSC resources on the [PowerShellGallery](https://www.powershellgallery.com/packages)!
+We're working hard right now to pull the prototype across the last couple of miles to get an MVP for testing - if you're interested in beta testing this project with us, please reach out on [slack in #windows](https://slack.puppet.com/) or via [email](ia-content@puppet.com)
+
+## Litmus Progress
+
+On [Litmus](https://github.com/puppetlabs/puppet_litmus), the team had a productive week. We managed to fix some bugs that you can check it [here](https://github.com/puppetlabs/puppet_litmus/pull/271) and [here](https://github.com/puppetlabs/puppet_litmus/pull/272). Also, now we are able to capture more informations for
```
Comment (sanfrancrisko) [inline _posts/2020-03-27-status-update.md]:
```suggestion
On [Litmus](https://github.com/puppetlabs/puppet_litmus), the team had a productive week. We managed to fix some bugs that you can check out [here](https://github.com/puppetlabs/puppet_litmus/pull/271) and [here](https://github.com/puppetlabs/puppet_litmus/pull/272). Also, now we are able to capture more information for 
```

File: _posts/2020-03-27-status-update.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+A bit further out and we'll have a new namespace on the forge - dsc - where we'll publish these automatically generated modules as 1:1 matches for the PowerShell modules with DSC resources on the [PowerShellGallery](https://www.powershellgallery.com/packages)!
+We're working hard right now to pull the prototype across the last couple of miles to get an MVP for testing - if you're interested in beta testing this project with us, please reach out on [slack in #windows](https://slack.puppet.com/) or via [email](ia-content@puppet.com)
+
+## Litmus Progress
+
+On [Litmus](https://github.com/puppetlabs/puppet_litmus), the team had a productive week. We managed to fix some bugs that you can check it [here](https://github.com/puppetlabs/puppet_litmus/pull/271) and [here](https://github.com/puppetlabs/puppet_litmus/pull/272). Also, now we are able to capture more informations for 
+[honeycomb](https://github.com/puppetlabs/puppet_litmus/pull/264). 
+
```
Comment (sanfrancrisko) [inline _posts/2020-03-27-status-update.md]: Maybe also add this line about our progress with the core module Litmus ports:
```suggestion
We only have two of our core modules left to convert to Litmus. Work has begun on [puppetlabs-reboot](https://forge.puppet.com/puppetlabs/reboot), with [puppetlabs-ibm_installation_manager](https://forge.puppet.com/puppetlabs/ibm_installation_manager) being the last module on the list! If you’re interesting in converting a module to Litmus, feel free to get in touch with us during [Office Hours](https://puppet.com/community/office-hours/)
```

Comment (sheenaajay) [reply _posts/2020-03-27-status-update.md]: @daianamezdrea Could you please add the following modules too
Work has begun on [puppetlabs-websphere_application_server](https://forge.puppet.com/puppetlabs/websphere_application_server) and [puppetlabs-satellite_pe_tools](https://forge.puppet.com/puppetlabs/satellite_pe_tools)

Comment (daianamezdrea) [reply _posts/2020-03-27-status-update.md]: Sure, @sheenaajay , thank you!

**Comment by daianamezdrea** [comment]
https://github.com/puppetlabs/iac/pull/36

**Pull request closed without merging**